### DISCUSSION TITLE
Openssl 3.2.0 compile for 64bit.

### DISCUSSION THREAD

**Discussion by hicham65** [openssl/openssl#23790] [Q&A] [answered]
which command shall I use to generate a 64-bit openssl library?
Below is the configure command I used.

            ./Configure solaris-sparcv9-cc \
            --prefix=$openssl_install_dir \

Thanks

**Comment by nhorman** [comment, marked as answer]
run
```
./Configure help
```
It will list the available targets
solaris-sparcv9-cc I believe targets 32 bit sparc, if you want 64 bit you probably want solaris64-sparcv9-cc

**Comment by hicham65** [comment]
thanks, that worked fine.
